Library/BookEntities/Feedback.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Library.BookEntities
+{
+    public class Feedback
+    {
+        public Guid FeedbackId { get; set; }
+        public string ReviewerName { get; set; }
+        public string? Comment { get; set; }
+        public int Rating { get; set; }
+        public DateTime Date { get; set; }
+    }
```

Comment:
Could be also added a new property Purchased : bool, which will indicate for all users that this comment left by someone who already bought this book.

Reply:
> Could be also added a new property Purchased : bool, which will indicate for all users that this comment left by someone who already bought this book.

done